pkg/opensearch/snapshot_tests/lucene_trace_spans_test.go

```diff
@@ -0,0 +1,45 @@
+package snapshot_tests
+
+import (
+	"context"
+	"io"
+	"net/http"
+	"testing"
+
+	"github.com/grafana/grafana-plugin-sdk-go/backend"
+	"github.com/grafana/opensearch-datasource/pkg/opensearch"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func Test_trace_spans_request(t *testing.T) {
```

Comment:
should there be a response test?

Reply:
Sadly I couldn't get the response test to work in snapshot because the response processing uses maps, so the result has slightly different order every time. Snapshot tests fail for this reason. Will leave a comment though, good catch!